test/test_config.py

```diff
@@ -142,3 +144,132 @@ def test_error_gs_dataset_mismatch(self):
 
         with pytest.raises(ValueError):
             config.init_global(test_config)
+
+    def test_eval_ml_coupling(self):
```

Comment:
You can use `@pytest.mark.parametrize` to remove the redundant code. You will create test function arugments and then provide a list of values the test case should take. See https://github.com/Reed-CompBio/spras/blob/master/test/test_util.py#L36-L42 for an example.